### DISCUSSION TITLE
🀄Please tell me your favorite hand in mahjong

### DISCUSSION THREAD

**Discussion by kangy106** [NAIST-SE/AttractingContributors#1088] [2024 Attraction Contributions]
Click Here! https://github.com/kangy106/mahjong/blob/main/README.md

**Comment by KazumaYamasaki** [comment]
Added `Ryuiso` to your README.md
I would appreciate it if you could also make a pull request to [my repository](https://github.com/KazumaYamasaki/AttractingContributors-Keyboard) !
↓
https://github.com/KazumaYamasaki/AttractingContributors-Keyboard
